Resources/config/routing_api.yml

```diff
@@ -13,9 +13,11 @@ sulu_form.forms:
 
 sulu_form.dynamic:
     type: rest
-    resource: Sulu\Bundle\FormBundle\Controller\DynamicController
+    resource: sulu_form.dynamic_controller
     prefix: form
     name_prefix: sulu_form.
+    options:
+        expose: true
```

Comment:
What is this doing? :thinking:

Reply:
As far as I understand it, this is needed to make the routes for the `DynamicController` available. 
If I remove this options, I get an error that the route doesn't exist.

Reply:
Ah, ok, so this is probably from the `FOSJsRoutingBundle`... I think in order to keep stuff consistent, I would choose the same way as it was done using the [`sulu_admin.metadata` route](https://github.com/sulu/sulu/pull/4671/files#diff-552d74d7863d5ff3781de14203825633R102).